frontend/app/routes/protected/renew/$id/confirm-email.tsx

```diff
@@ -98,12 +100,33 @@ export async function action({ context: { appContainer, session }, params, reque
     params,
     request,
     session,
-    state: { contactInformation: { ...state.contactInformation, ...parsedDataResult.data } },
+    state: { emailPreValidation: parsedDataResult.data },
+    // state: { contactInformation: { ...state.contactInformation, ...parsedDataResult.data } },
   });
 
   const idToken: IdToken = session.get('idToken');
   appContainer.get(TYPES.domain.services.AuditService).createAudit('update-data.renew.confirm-email', { userId: idToken.sub });
 
+  if (parsedDataResult.data.email && parsedDataResult.data.email !== state.contactInformation?.email) {
+    // Create a new verification code and store the code in session
+    const verificationCode = verificationCodeService.createVerificationCode('anonymous');
```

Comment:
Just noticed this should be changed for all `protected` pages (can be done in another PR - will add an ADO work item):
```suggestion
    const verificationCode = verificationCodeService.createVerificationCode(idToken.sub);
```

Reply:
since I picked up the ADO ticket, I'll address in a separate PR